.travis.yml

```diff
@@ -26,7 +26,7 @@ before_script:
 
 # command to run tests, e.g. python setup.py test
 script:
-  - tox -e $TOX_ENV
+  - tox -e $TOX_ENV -r
```

Comment:
Not for travis though? It doesn't cache anything. The intention was to make travis speedier by having it reuse stuff :)